src/Client.php

```diff
@@ -1,8 +1,8 @@
 <?php
 
-namespace pusher;
+namespace Pusher;
 
-use pusher\Version;
+use Pusher\Version;
```

Comment:
@zimbatm It is used in userAgent(), however both classes are in the same namespace so it is not strictly needed.